.travis.yml

```diff
@@ -17,12 +17,19 @@ install:
   - make install-ci
 
 services:
+  - docker
   - postgresql
   - redis
 
 before_script:
+  - docker run -d -p 8000:8000 dwmkerr/dynamodb
```

Comment:
official image would be betterr: https://hub.docker.com/r/amazon/dynamodb-local/

Reply:
Thank you for the suggestion!
I've updated the PR.